src/System.Drawing.Common/src/System.Drawing.Common.csproj

```diff
@@ -34,10 +35,12 @@
     <Compile Include="System\Drawing\ColorTranslator.cs" />
     <Compile Include="System\Drawing\ContentAlignment.cs" />
     <Compile Include="System\Drawing\IDeviceContext.cs" />
-    <Compile Include="System\Drawing\Image.cs" />
     <Compile Include="System\Drawing\GdiplusNative.cs" />
     <Compile Include="System\Drawing\Graphics.cs" />
     <Compile Include="System\Drawing\GraphicsUnit.cs" />
+    <Compile Include="System\Drawing\Image.cs" />
+    <Compile Include="System\Drawing\Image.NotSerializable.cs" />
```

Comment:
nit: we use ascii ordering here, Image.NotSerializable.cs should come before Image.cs

Reply:
and Icon before Image.